municipal_finance/settings.py

```diff
@@ -50,6 +52,11 @@
     'corsheaders',
 )
 
+# Sites
+if DEBUG:
+    SITE_ID = 2  # Scorecard
```

Comment:
In production, the sites framework uses the domain of the incoming request to work out what site is active. In development, we don't have a domain and this setting tells Django which site to use. The ID is the id of the corresponding Site object in the database.

What, random integers in settings files aren't your thing? :)